sb1-scope.json

```diff
@@ -0,0 +1,173 @@
+          "file": "\\/*\\/finansiering\\/rest\\/soknad\\/*",
+          "host": "^*.sparebank1.no$",
+          "protocol": "any"
+        },
+        {
+          "enabled": true,
+          "file": "\\/*\\/nettbank-privat\\/barn\\/rest\\/*",
+          "host": "^*.sparebank1.no$",
+          "protocol": "any"
+        },
+        {
+          "enabled": true,
+          "file": "\\/*\\/nettbank-privat\\/betaling\\/rest\\/payments\\/straksbetaling",
+          "host": "^*.sparebank1.no$",
+          "protocol": "any"
+        },
+        {
+          "enabled": true,
+          "file": "\\/*\\/nettbank-privat\\/betaling\\/rest\\/overforing-fra-kredittkort",
+          "host": "^*.sparebank1.no$",
+          "protocol": "any"
+        },
+        {
+          "enabled": true,
+          "file": "\\/*\\/nettbank-privat\\/kort\\/rest\\/cards\\/credit\\/application*",
+          "host": "^*.sparebank1.no$",
+          "protocol": "any"
+        },
+        {
+          "enabled": true,
+          "file": "\\/*\\/nettbank-privat\\/kort\\/rest\\/cards\\/credit\\/limit*",
+          "host": "^*.sparebank1.no$",
+          "protocol": "any"
+        },
+        {
+          "enabled": true,
+          "file": "\\/*\\/nettbank-privat\\/kort\\/rest\\/cards\\/credit\\/transactions",
+          "host": "^*.sparebank1.no$",
+          "protocol": "any"
+        },
+        {
+          "enabled": true,
+          "file": "\\/*\\/nettbank-privat\\/kort\\/rest\\/cards\\/credit\\/*",
+          "host": "^*.sparebank1.no$",
+          "protocol": "any"
+        },
+        {
+          "enabled": true,
+          "file": "\\/*\\/nettbank-privat\\/kort\\/rest\\/cards\\/debit",
+          "host": "^*.sparebank1.no$",
+          "protocol": "any"
+        },
+        {
+          "enabled": true,
+          "file": "\\/*\\/nettbank-privat\\/kort\\/rest\\/cards\\/debit\\/*order*",
+          "host": "^*.sparebank1.no$",
+          "protocol": "any"
+        },
+        {
+          "enabled": true,
+          "file": "\\/*\\/nettbank-privat\\/kort\\/rest\\/cards\\/*\\/fraud",
+          "host": "^*.sparebank1.no$",
+          "protocol": "any"
+        },
+        {
+          "enabled": true,
+          "file": "\\/*\\/nettbank-privat\\/kort\\/rest\\/cards\\/*\\/pin",
+          "host": "^*.sparebank1.no$",
+          "protocol": "any"
+        },
+        {
+          "enabled": true,
+          "file": "\\/*\\/nettbank-privat\\/sparing\\/rest\\/splitt\\/*",
+          "host": "^*.sparebank1.no$",
+          "protocol": "any"
+        },
+        {
+          "enabled": true,
+          "file": "\\/*\\/nettbank-privat\\/sparing\\/rest\\/aksjetjeneste",
+          "host": "^*.sparebank1.no$",
+          "protocol": "any"
+        },
+        {
+          "enabled": true,
+          "file": "\\/*\\/nettbank-privat\\/sparing\\/rest\\/aksjetjeneste\\/ask",
+          "host": "^*.sparebank1.no$",
+          "protocol": "any"
+        },
+        {
```

Comment:
looks to be a duplicate of line 139-143

Reply:
Thanks! e2da58b93d5542a31353b248c9dbf1c1ce242a75